rust/src/smb/smb.rs

```diff
@@ -1511,6 +1544,14 @@ impl SMBState {
 
         match parse_nbss_record_partial(input) {
             Ok((output, ref nbss_part_hdr)) => {
+
+                let _nbss_pdu = applayer_new_record_tc(flow, base, input, nbss_part_hdr.length as i32 + 4, SMBPduType::NBSS as u8);
```

Comment:
Why would the parser need a "handle" for the record?

Reply:
To update it later to:
- set the tx_id
- set an event if the parser encounters an exception in parsing the record